src/dao/MoodDao.java

```diff
@@ -31,7 +41,41 @@ public MoodDao() {
      * executing the query
      */
     public ArrayList<Mood> getAllMoods() throws DaoException {
-        return null;
+        BufferedReader reader = null;
+        try {
+            ArrayList<Mood> moods = new ArrayList<Mood>();
+            String ln = "";
+
+            reader = new BufferedReader(new FileReader("data/Moods.csv"));
+            reader.readLine();
```

Comment:
This should only be here if we have a header. If not, please remove or we'll skip the first mood in the file.